dstep/Configuration.d

```diff
@@ -18,42 +18,71 @@ struct Configuration
     /// array of file names to translate to D
     string[] inputFiles;
 
-    /// expected programming language of input files
+    @("language|x", "Treat subsequent input files as having type <language>.")
     Language language;
 
+    @("version", "Show dstep version.")
+    bool dstepVersion;
+
+    @("clang-version", "Show libclang version.")
+    bool clangVersion;
+
     /// array of parameters needed to be forwarded to clang driver
     string[] clangParams;
 
     /// output file name or folder (in case there are many input files)
     string output;
 
-    /// strip all comments while translating
-    bool noComments;
-
-    /// package name
```

Comment:
Now we've lost the documentation, if we would generate it in the future. Should be possible to have both?